mnamer/setting_store.py

```diff
@@ -218,6 +218,16 @@ class SettingStore:
             help="--episode-format: set episode renaming format specification",
         )(),
     )
+    relocation_mode: Optional[Union[RelocateType,str]] = dataclasses.field(
```

Comment:
Not sure why we must allow items of type "RelocateType". But i see that was done elsewhere. I think this is redundant because we only pass instances of str and never of RelocateType.